src/Test.FunctionalTests.Sarif/TestData/Multitool/BaselineOption/ExpectedOutputs/TEST1001.ValidateWithBaseline.sarif

```diff
@@ -23,6 +23,177 @@
               },
               "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
             },
+            {
+              "id": "ADO1015",
```

Comment:
>"id" [](http://example.com/codeflow?start=14&length=4)

for files changed in this folder,
my understanding is baseline is good for 
later when dev change code and baseline becomes different, that force the dev to look why and if it is as expected. 

This will be the first apply your new rules to the baseline.
You may want to browse through it they are as expected per the logic you got from the requirements.
Or we can also check this after we have created per rule test files.